### DISCUSSION TITLE
Videos/Animations

### DISCUSSION THREAD

**Discussion by Qtakruri** [RudolphRiedel/FT800-FT813#127] [Q&A]
Hey Rudolph,

I have some questions about playing videos or animations using your library. I want to play short clips (3-5) seconds at certain interrupts.

i was able to display pictures after using EVE Asset builder and compressing the files then converting them to C-compatible format. 
when it comes to animations,  what are the steps needed to display a short clip?  and how do you manage the memory here ? can i use a SD card to read from ? 

Thanks,

**Comment by RudolphRiedel** [comment]
Hi,

this is simply something that I have not explored so far.
For the most part, video or animations is out of scope for me.

Heck, I am struggeling right now to find an image to use for an icon. :-)
And how it looks seems to be a minor detail at this point. :-)

I would probaly try to go with an animation as the compression should be better than with a video.
Or I would go with a bunch of images and display them sequentially.
